spec/services/publish_draft_edition_service_spec.rb

```diff
@@ -39,6 +39,13 @@
       end
     end
 
+    it "updates the documents live edition to be the current edition" do
```

Comment:
It's outside the context because this applies whether a live edition exists or not. I can see how the `expect(document.live_edition).to be_nil` might create the impression that this belongs in that context though.

I might be able to change it to to: 

```
document = create(:document, :with_current_edition)
expect { described_class.call(document.current_edition, user, with_review: true) }
  .to change(document, :live_edition).to(document.current_edition)
```

does that help?

Reply:
Yes! You're right, it was the `nil` check that threw me.

Reply:
Great - I've made both code changes and grammar changes now.